src/script/tracking/EventTrackingRepository.ts

```diff
@@ -226,24 +231,27 @@ export class EventTrackingRepository {
   private async startProductReporting(trackingId?: string): Promise<void> {
     // This is a global object provided by the countly.min.js script
     if (!isCountlyLoaded()) {
+      this.countlyLogger.warn('Countly is not available');
       return;
     }
 
     if (!isCountlyEnabledAtCurrentEnvironment() || this.isProductReportingActivated) {
+      this.countlyLogger.warn('Countly is not enabled at this environment');
       return;
     }
     this.isProductReportingActivated = true;
 
-    const {isProduction} = getWebEnvironment();
```

Comment:
Maybe we keep this for peace of mind as a condition in addition to the nev env var?

Reply:
I dont believe we need that. 

This was just added because we didnt had an env variable.

Reply:
All good then